02-microarray/pathway-analysis_microarray_03_gsea.Rmd

```diff
@@ -437,27 +452,34 @@ Let's look for the gene set with the highest negative NES.
 
 ```{r}
 gsea_result_df %>%
-  # The `dplyr::arrange()` function sorts the values in ascending order
-  dplyr::arrange(NES)
+  # Use `top_n()` to return the top n observation using `NES` as the ordering
+  # variable -- note the use of `dplyr::desc()` to sort the values in descending order
+  dplyr::top_n(n = 1, wt = dplyr::desc(NES))
 ```
 
-The gene set `HALLMARK_E2F_TARGETS` has a highly negative NES.
+The gene set `HALLMARK_E2F_TARGETS` has the highest negative NES.
```

Comment:
```suggestion
The gene set `HALLMARK_E2F_TARGETS` has the most negative NES.
```

It didn't occur to me before but "highest negative" sounds weird. Should we change this or is it not as weird as I'm thinking it is?

Reply:
Agreed, for consistency, I mimicked this language when speaking about the _most_ positive NES as well. Please let me know if you think I should change that back to _highest positive_.